coderd/database/queries/notifications.sql

```diff
@@ -19,7 +19,7 @@ VALUES (@id,
         @payload::jsonb,
         @targets,
         @created_by)
-RETURNING *;
+RETURNING id;
```

Comment:
I think you can revert the hack if you change this to `RETURNING id::uuid;`. Alternatively, in `VALUES(`, using `@id::uuid` could also work.

Reply:
Actually, I may be wrong. Not really sure what you ran into 😄

Reply:
Oh nice! That worked 🥇